components/todo/TodoItem.vue

```diff
@@ -1,23 +1,47 @@
 <template>
   <div class="todo-item">
-    <div class="todo-title">{{ todo.title }}</div>
-    <div>
-      <div class="created-at">
-        {{ $t('created-at') }}:
+    <div class="todo-item__header">
+      <span
+        class="todo-title"
+        :class="todo.completedAt ? 'text-line-through' : ''"
+        >{{ todo.title }}</span
+      >
+      <span class="time"
+        >{{ $t('created-at') }}:
         {{ format(todo.createdAt, 'yyyy-MMM-dd') }}
-      </div>
-      <div>
-        <button class="todo-action">{{ $t('complete') }}</button>
-        <button class="todo-action">{{ $t('edit') }}</button>
-        <button class="todo-action" @click.prevent="deleteTodo">
-          {{ $t('delete') }}
-        </button>
+      </span>
+    </div>
+
+    <div class="todo-item__footer">
+      <div class="action-buttons">
+        <div>
+          <button
+            v-if="!todo.completedAt"
```

Comment:
We can create a computed property named `isTodoCompleted` and use it here. Avoiding negation is always a good idea IMHO.

Reply:
Resolved